components/brave_new_tab_ui/containers/newTab/settings/backgroundImage.tsx

```diff
@@ -21,16 +22,34 @@ import { Toggle } from '../../../components/toggle'
 
 import { getLocale } from '../../../../common/locale'
 
+import SolidColorChooser from './solidColorChooser'
+
 interface Props {
   toggleBrandedWallpaperOptIn: () => void
   toggleShowBackgroundImage: () => void
   useCustomBackgroundImage: (useCustom: boolean) => void
+  useSolidColorBackground: (color: string) => void
   brandedWallpaperOptIn: boolean
   showBackgroundImage: boolean
   featureCustomBackgroundEnabled: boolean
 }
 
-class BackgroundImageSettings extends React.PureComponent<Props, {}> {
+interface State {
+  location: string
```

Comment:
Let's strongly type this so it's obvious what the options are, and we can catch errors during compile. Maybe an enum, e.g. `type BackgroundView = { LIST, COLORS }`, or keep it a `string` and simply `type BackgroundView = '' | 'solidColors'`, or even simpler since we only have 1 child item, `isViewingColors: boolean`. But I understand if you want to optimize for the future with either first or second option.